x-pack/plugins/cloud_security_posture/public/components/no_findings_states.tsx

```diff
@@ -125,11 +171,20 @@ export const NoFindingsStates = () => {
     options: { refetchInterval: REFETCH_INTERVAL_MS },
   });
   const status = getSetupStatus.data?.status;
+  const indicesStatus = getSetupStatus.data?.indicesDetails;
+  const unprivilegedIndices =
+    indicesStatus &&
+    indicesStatus
+      .filter((idxDetails: IndexDetails) => idxDetails.status === 'unprivileged')
```

Comment:
_nit_
```suggestion
      .filter((idxDetails) => idxDetails.status === 'unprivileged')
```

`IndexDetails` should be inferred